tests/models/fuyu/test_modeling_fuyu.py

```diff
@@ -278,9 +278,8 @@ def setUp(self):
     @slow
     @require_torch_gpu
     def test_model_8b_chat_greedy_generation_bus_captioning(self):
-        EXPECTED_TEXT_COMPLETION = """A bus parked on the side of a road.|ENDOFTEXT|"""
+        EXPECTED_TEXT_COMPLETION = """A blue bus parked on the side of a road.|ENDOFTEXT|"""
```

Comment:
The generation changed after some refactoring of the image processor. Testing I was able to confirm it's caused by small floating point changes that happen when rescaling the pixel values by 1/255. Previously, the pixel were divided by 255, whereas they are now multiplied by `1 / 255`.